todo_app/data/session_items.py

```diff
@@ -2,6 +2,19 @@
 
 import requests, os
 from datetime import datetime
+from pymongo import MongoClient
+from bson.objectid import ObjectId
+
+db=None
```

Comment:
Instead of having a global variable you could refactor this into a class with `db` as a field - that way you can use `self.db` instead of doing
```python
global db
if(db==None):
    db=OpenMongo()
```
each time, though either works